release/ray_release/test.py

```diff
@@ -56,6 +57,13 @@ def get_byod_image_tag(self) -> str:
             os.environ["BUILDKITE_COMMIT"],
         )
         ray_version = commit[:6]
+        branch = os.environ.get("BUILDKITE_BRANCH", "")
+        assert branch == "master" or branch.startswith(
+            "releases"
+        ), f"Invalid branch name {branch}"
+        if branch.startswith("releases"):
+            release_name = re.search("[0-9]+\.[0-9]+\.[0-9].*", branch).group(0)
```

Comment:
do not quite understand why this needs a regex..

isn't this just striping the `"releases/"` prefix? what will happen it is not matching?

Reply:
w00t, my bad, this change comes from a wrong base. The other PR explains why, but basically I keep it consistent with how the logic is in ci/build/build-docker-images.py